src/adapters/MembershipJSONAdapter.test.js

```diff
@@ -0,0 +1,56 @@
+import memberships from '../data/memberships';
```

Comment:
Thanks for adding missing tests! 🙂
I think you followed the format on the people adapter tests, which is a great start, but those tests where written very early on. Using the [`MeetingsJSONAdapter` tests](https://github.com/webex/components/blob/master/src/adapters/MeetingsJSONAdapter.test.js) is a better guide of where we want to take the tests, in terms of what we test and how we word our test descriptions